contrib/pkg/broker/user_provided/controller/controller.go

```diff
@@ -66,9 +66,15 @@ func (c *userProvidedController) Catalog() (*brokerapi.Catalog, error) {
 					ID:          "86064792-7ea2-467b-af93-ac9694d96d52",
 					Description: "Sample plan description",
 					Free:        true,
+				}, {
+					Name:        "premium",
+					ID:          "cc0d7529-18e8-416d-8946-6f7456acd589",
+					Description: "Premium plan",
+					Free:        true,
```

Comment:
I wonder if this should be "false" just so we can test with a non-free plan too?

Reply:
It could be. I don't know that it matters much since that field is only consumable by the user and not used in our code for anything. But it won't hurt, so I'm happy to make that change

Reply:
thanks